agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/SecondEntryPoint.java

```diff
@@ -349,6 +349,10 @@ private static Set<Feature> initStatsbeatFeatureSet(Configuration config) {
     if (!config.preview.sampling.overrides.isEmpty()) {
       featureList.add(Feature.PREVIEW_SAMPLING);
     }
+    // TODO (heya) track sampling overrides GA feature after updating the standalone exporter
+    //    if (!config.sampling.overrides.isEmpty()) {
+    //      featureList.add(Feature.SAMPLING);
+    //    }
```

Comment:
can go ahead and send PR to azure-sdk-for-java repo to rename `PREVIEW_SAMPLING` -> `SAMPLING_OVERRIDES`

Reply:
```suggestion
```